### PR TITLE
Refactor/remove instance3

### PR DESCRIPTION
**Pull request by sudden6**
- [x] Commits follow our [git commit guidelines](https://github.com/qTox/qTox/blob/master/CONTRIBUTING.md#git-commit-guidelines)

---
This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/6141)

### CODE REVIEW

File: src/net/avatarbroadcaster.cpp [hunk 1]
```diff
@@ -79,8 +82,10 @@ void AvatarBroadcaster::sendAvatarTo(uint32_t friendId)
  */
 void AvatarBroadcaster::enableAutoBroadcast(bool state)
 {
-    QObject::disconnect(autoBroadcastConn);
-    if (state)
-        autoBroadcastConn =
-            QObject::connect(Core::getInstance(), &Core::friendStatusChanged, autoBroadcast);
+    this->disconnect(&core, nullptr, this, nullptr);
+    if (state) {
+            connect(&core, &Core::friendStatusChanged, [=](uint32_t friendId, Status::Status) {
```
Comment (Diadlo) [inline src/net/avatarbroadcaster.cpp]: Extra indent

Comment (sudden6) [reply src/net/avatarbroadcaster.cpp]: Thx, fixed.

**Review by anthonybilinski — APPROVED** [review]
:lgtm_strong: 

> Reviewed 3 of 7 files at r1, 4 of 4 files at r3.
> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/6141#-:-M71kIaqCxG7kl0y16u4:b-uwkowd)* status: :shipit: complete! 1 of 1 LGTMs obtained

___
*[src/net/avatarbroadcaster.cpp, line 89 at r3](https://reviewable.io/reviews/qtox/qtox/6141#-M71jMZc6Bq7a2RbPBwI:-M71jMZc6Bq7a2RbPBwJ:b-6w6buz) ([raw file](https://github.com/qtox/qtox/blob/c2a95f0a18203e9644677a5a89ab2e184da84cb0/src/net/avatarbroadcaster.cpp#L89)):*
> ```cpp
>         connect(&core, &Core::friendStatusChanged, [=](uint32_t friendId, Status::Status) {
>             this->sendAvatarTo(friendId);}
>         );
> ```

move `}` down a line?

**Review by sudden6 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/6141)* status: :shipit: complete! 1 of 1 LGTMs obtained

___
*[src/net/avatarbroadcaster.cpp, line 89 at r3](https://reviewable.io/reviews/qtox/qtox/6141#-M71jMZc6Bq7a2RbPBwI:-M71qn-b69kK3QfHUvxr:b-jxsber) ([raw file](https://github.com/qtox/qtox/blob/c2a95f0a18203e9644677a5a89ab2e184da84cb0/src/net/avatarbroadcaster.cpp#L89)):*
<details><summary><i>Previously, anthonybilinski (Anthony Bilinski) wrote…</i></summary><blockquote>

move `}` down a line?
</blockquote></details>

ran code formatter

**Review by anthonybilinski — APPROVED** [review]
:lgtm_strong: 

> Reviewed 3 of 3 files at r4.
> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/6141#-:-M71rDcS3qoyK9km8ASD:b-uwkowd)* status: :shipit: complete! 1 of 1 LGTMs obtained

**Pull request merged**